### PR TITLE
Added emadb bio

### PR DESCRIPTION
**Pull request by emadb**

### CODE REVIEW

File: mentors/elixir/e.json [hunk 1]
```diff
@@ -1 +1,8 @@
-[]
+[{
```
Comment (rpottsoh) [inline mentors/elixir/e.json]:
```suggestion
[
  {
```

File: mentors/elixir/e.json [hunk 2]
```diff
@@ -1 +1,8 @@
...
+  "github_username": "emadb",
+  "name": "Emanuele DelBono",
+  "link_text": null,
+  "link_url": null,
+  "avatar_url": null,
+  "bio": "I'm a developer passionate about clean code that falled in love with Elixir"
+}]
```
Comment (rpottsoh) [inline mentors/elixir/e.json]:
```suggestion
  }
]
```

File: mentors/elixir/e.json [hunk 3]
```diff
@@ -1 +1,8 @@
-[]
+[{
+  "github_username": "emadb",
+  "name": "Emanuele DelBono",
+  "link_text": null,
+  "link_url": null,
+  "avatar_url": null,
+  "bio": "I'm a developer passionate about clean code that falled in love with Elixir"
```
Comment (rpottsoh) [inline mentors/elixir/e.json]: indent lines 2-7 by two spaces so they are two spaces indented in from `{` and `}`.

Comment (rpottsoh) [inline mentors/elixir/e.json]:
```suggestion
  "bio": "I'm a developer passionate about clean code that fell in love with Elixir."
```

Comment (emadb) [reply mentors/elixir/e.json]: ops! :-)

**Comment by rpottsoh** [comment]
Please correct the issues that I have indicated by submitting another commit to this PR.

**Comment by rpottsoh** [comment]
Perfect, thanks @emadb.  Welcome to the Elixir track.

**Pull request merged**